packages/components/src/index.js

```diff
@@ -16,6 +16,11 @@ export {
 	default as Animate,
 	getAnimateClassName as __unstableGetAnimateClassName,
 } from './animate';
+export {
+	__unstableMotion,
+	__unstableAnimatePresence,
+	__unstableAnimateSharedLayout,
```

Comment:
Both methods aren't used outside of `@wordpress/components` so in practice we don't need to expose them for now. It won't have an impact on the bundle size though since they are in use internally.

Reply:
Sounds good, updated in https://github.com/WordPress/gutenberg/pull/33717/commits/63520aa5dc45f9b774aad47ece74f318d6ac5c35